From 43c67ae1af8212e8aea5055455884384ef4e40cb Mon Sep 17 00:00:00 2001
From: Andrew Sutherland <dr3wsuth3rland@gmail.com>
Date: Sun, 28 Apr 2013 23:26:50 -0500
Subject: [PATCH] apache changes

---
 code.py | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/code.py b/code.py
index c814101..2102ff4 100755
--- a/code.py
+++ b/code.py
@@ -3,10 +3,10 @@
 import web
 
 # apache hax for local imports
-#import os, sys
-#abspath = os.path.dirname(__file__)
-#sys.path.append(abspath)
-#os.chdir(abspath)
+import os, sys
+abspath = os.path.dirname(__file__)
+sys.path.append(abspath)
+os.chdir(abspath)
 # local imports
 from devices import *
 from operations import find_builds, get_screenshots, search_files, find_logs
@@ -144,9 +144,9 @@ class Catchall:
         raise web.seeother('/404/')
 
 # lighttpd
-if __name__ == "__main__":
-    app = web.application(urls, globals())
-    app.run()
+#if __name__ == "__main__":
+#    app = web.application(urls, globals())
+#    app.run()
 
 # apache2 + wsgi
-#application = web.application(urls, globals()).wsgifunc()
+application = web.application(urls, globals()).wsgifunc()
-- 
1.8.2.1

